src/language/pattern-types.md

```diff
@@ -142,6 +152,10 @@ A logical-or pattern separates subpatterns by `||` and matches if any of the
 branches match. Branches are evaluated left-to-right. Once a branch matches, the
 rest are not evaluated.
 
+Logical-or patterns are useful for having multiple cases share a body in [switch
+expressions or statements][].
```

Comment:
It looks you introduced this during your merge, but as far as I can tell, this link isn't defined.

Broken link definitions will be caught after https://github.com/dart-lang/site-www/pull/4900 lands :)

Reply:
Ah that's left over somehow from when I moved that comment to the patterns page. Thanks for catching!